packages/swapkit/core/src/client/index.ts

```diff
@@ -635,6 +635,9 @@ export class SwapKitCore<T = ''> {
   connectWalletconnect = async (_chains: Chain[], _options?: any): Promise<void> => {
     throw new SwapKitError('core_wallet_walletconnect_not_installed');
   };
+  connectKeepkey = async (_chains: Chain[], _derivationPath: number[][]): Promise<string> => {
```

Comment:
Change return type

Reply:
Pairing keys are UUID v4() strings that are always 36 characters in length. These pairing keys ensure that users cannot be phished by fake dApps that pretend to be ThorSwap or previously approved dApps.